### ISSUE TITLE
Baseline-to-baseline advice

### ISSUE THREAD

**Issue by skef** [fontforge/fontforge#3610]
Is [this advice](http://fontforge.github.io/editexample5.html#baseline) about baseline-to-baseline spacing in OpenType and TrueType still accurate?

**Comment by ctrlcctrlv** [comment]
No. I know that the Linux section is wrong, and would definitely assume the Mac is as well.

**Comment by skef** [comment]
It would probably better to reorganize this into a short list of whatever conventions are in wide use, and then some brief, vague statements about where each might be used. 

So are there other conventions not currently described?

**Comment by ctrlcctrlv** [comment]
The conventions described are of course correct, but I'm highly skeptical of their relevance to modern platforms. Especially `On linux/unix    I doubt there is any standard behavior.` is dubious, given FreeType+Pango's ubiquity on Linux.

**Comment by skef** [comment]
Mentioning [this](https://gitlab.gnome.org/GNOME/gtk/issues/1626) for reference.

**Comment by skef** [comment]
More on FreeType:

> `FT_Face.height`: This value is the vertical distance between two consecutive baselines, expressed in font units. It is always positive. Only relevant for scalable formats. If you want the global glyph height, use `ascender - descender`.

Hmm, but how do they calculate it?
